src/Microsoft.Graph.Auth/Microsoft.Graph.Auth.csproj

```diff
@@ -6,30 +6,34 @@
     <AssemblyVersion>0.1.0</AssemblyVersion>
     <Authors>Microsoft</Authors>
     <Company>Microsoft Corporation</Company>
-    <TargetFrameworks>netstandard1.3;net45;</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
     <AssemblyName>Microsoft.Graph.Auth</AssemblyName>
     <PackageId>Microsoft.Graph.Auth</PackageId>
     <PackageTags>Microsoft Office365;Graph;GraphServiceClient;Outlook;OneDrive;AzureAD;GraphAPI;Productivity;SharePoint;Intune;SDK;MSAL</PackageTags>
-    <PackageReleaseNotes></PackageReleaseNotes>
+    <PackageReleaseNotes>
+      - Update all authentication providers to use MSAL V3+.
+      - Update UsernamePasswordProvider to use SecureString for passwords.
+      - Remove ITokenStorageProvider interface.
+    </PackageReleaseNotes>
     <PackageProjectUrl>https://developer.microsoft.com/graph</PackageProjectUrl>
     <PackageLicenseUrl>http://aka.ms/devservicesagreement</PackageLicenseUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <SignAssembly>false</SignAssembly>
     <DelaySign>true</DelaySign>
     <!--Skip 1.13.0-->
-    <VersionPrefix>0.1.0</VersionPrefix>
-    <VersionSuffix></VersionSuffix>
+    <VersionPrefix>0.2.0</VersionPrefix>
```

Comment:
https://www.nuget.org/packages/Microsoft.Graph.Auth/0.1.0-preview.2

VersionPrefix should be 0.1.0
VersionSuffix should be preview.3